products/grafana.md

```diff
@@ -13,7 +13,7 @@ identifiers:
 
 auto:
   methods:
-  -   git: https://github.com/grafana/grafana
+  -   github_releases: grafana/grafana
```

Comment:
Did you observe major differences between both methods ? Because git is preferred over github_releases because it's usually a bit faster.

Reply:
the major difference is about the release note, there is some time gap between 11.2.0 tag and 11.2.0 release  (it got tagged on Aug 21 but released on Aug 27)